easybuild/scripts/bootstrap_eb.py

```diff
@@ -854,6 +854,8 @@ def main():
     self_txt = open(__file__).read()
     if IS_PY3:
         self_txt = self_txt.encode('utf-8')
+    info("DEPRECATED: Use of the EasyBuild boostrap script is deprecated and may not work.")
+    info("Recommended install methods: https://docs.easybuild.io/en/latest/Installation.html\n")
```

Comment:
I would take this one step further, and abort the bootstrap unless an environment variable like `$EASYBUILD_BOOTSTRAP_DEPRECATED` is defined, to actively discourage still using it.

Reply:
see https://github.com/branfosj/easybuild-framework/pull/2